conf/maestro.config

```diff
@@ -0,0 +1,49 @@
+params {
+  config_profile_description = 'Institut Pasteur Maestro cluster profile'
+  config_profile_url = 'https://confluence.pasteur.fr/spaces/viewspace.action?key=FAQA'
```

Comment:
I suspect @maxulysse will suggest the following URL: https://research.pasteur.fr/en/equipment/maestro-compute-cluster/